### PR TITLE
Refactoring ValidateUsages for bootstrap tokens.

### PR DESCRIPTION
**Pull request by xiangpengzhao**
**What this PR does / why we need it**:
Refactoring and cleanup.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/cc @luxas @jbeda 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/bootstrap/api/types.go [hunk 1]
```diff
@@ -95,3 +95,6 @@ const (
 	// tokens (in addition to any groups from BootstrapTokenExtraGroupsKey).
 	BootstrapDefaultGroup = "system:bootstrappers"
 )
+
+// DefaultTokenUsages specifies the default functions a token will get.
+var DefaultTokenUsages = []string{"signing", "authentication"}
```
Comment (luxas) [inline pkg/bootstrap/api/types.go]: I'd call this `KnownTokenUsages` instead. This package doesn't assume defaults for how kubeadm should create the tokens.

File: cmd/kubeadm/app/constants/constants.go [hunk 2]
```diff
@@ -220,9 +219,6 @@ var (
 	// AuthorizationWebhookConfigPath defines the supported location of webhook config file
 	AuthorizationWebhookConfigPath = filepath.Join(KubernetesDir, "webhook_authz.conf")
 
-	// DefaultTokenUsages specifies the default functions a token will get
-	DefaultTokenUsages = []string{"signing", "authentication"}
```
Comment (luxas) [inline cmd/kubeadm/app/constants/constants.go]: this could point to `bootstrapapi.KnownTokenUsages`

File: pkg/bootstrap/api/types.go (hunk 1 above)
Comment (xiangpengzhao) [reply pkg/bootstrap/api/types.go]: Reasonable! :+1: 
updated.

**Review by luxas — APPROVED** [review]
/lgtm

**Comment by luxas** [comment]
/assign @jbeda 
for approval in top-level `pkg/` (goh, we should really make myself, @jbeda approvers in `pkg/bootstrap/api` and @mattmoyer a reviewer)

**Comment by xiangpengzhao** [comment]
/retest

@luxas I create a separate PR https://github.com/kubernetes/kubernetes/pull/57622 to add OWNERS file.

**Comment by xiangpengzhao** [comment]
I cherry-picked the OWNER commit here.

**Review by luxas — APPROVED** [review]
/lgtm

**Review by jbeda — APPROVED** [review]

**Pull request merged**